docs/applications_general.py

```diff
@@ -52,27 +42,24 @@
 # MRS: I'm not sold on using the name subgraph, because it doesn't
 #      describe what it is a subgraph of. Can this be made more specific?
 
-# MRS: can a name be used instead of 'P' below? 
-#
-
-subgraph = scalems.subgraph(variables={
+simulation_and_analysis_iteration = scalems.subgraph(variables={
         'conformation': initial_input,
-        'P': scalems.ndarray(0., shape=(N, N)),
+        'transition_matrix': scalems.ndarray(0., shape=(N, N)),  # number of simulations or number of clusters?
```

Comment:
Oops. Yes, here N means number of simulations, but seems to be used to set the number of cluster, too. I think the idea was that it would be convenient (after the first iteration) to be running one simulation per cluster at each iteration and then re-clustering the conformations collected from all simulations, then resampling from the new clusters for the next round of input.